hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -251,6 +252,12 @@ public void init(OzoneConfiguration configuration) throws IOException {
   @Override
   public Void call() throws Exception {
     if (ozoneConfiguration != null) {
+      if (ozoneConfiguration.getBoolean(HddsConfigKeys.HDDS_CONTAINER_PERSISTDATA,
```

Comment:
Is this required because you are added a test for freon with the null ChunkManager?

Reply:
this is required because each individual test case in TestDataValidate sets this parameter in the RandomKeyGenerator builder.